### PR TITLE
T-680: Update links; simplify based on user interview

### PR DESCRIPTION
**Pull request by pmillspaugh**
Depends on https://github.com/membrane-io/docs/pull/32

Close T-680
Close T-835
Close T-836
Close T-837

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -32,15 +32,15 @@ import { state } from "membrane";
 
 // We'll use `state` here to store your name and a count of the updates you've made to this program.
 state.name = "[add your name here]";
-state.updates ??= 0; // `??= 0` (nullish coalescing assignment) is equivalent to `= state.updates ?? 0`
```
Comment (pmillspaugh) [inline index.ts:35]: I think having a lesser known JS operator here is more confusing than cool

File: index.ts [hunk 2]
```diff
@@ -49,10 +49,8 @@ console.log(`Update count: ${state.updates}`);
  */
 
 // A Membrane program can export _actions_ that you invoke. This program exports a `run` action.
-// Let's invoke the `run` action, defined as a function below. To invoke an action, you can:
-// 1. Click on the "Invoke ▶️" button directly above the function signature
-// 2. In the Membrane Explorer, click "getting-started" then click "Invoke" to the right of "run"
-// 3. Use the Command+Enter keyboard shortcut
```
Comment (juancampa) [inline index.ts:55]: I think it's okay to remove this for now but we should fix it soon

File: index.ts [hunk 3]
```diff
@@ -62,13 +60,13 @@ export async function run() {
...
 // Try changing the string returned, hit save, and see your text in the Explorer next to `getting-started`.
-// `status` is a default field that all Membrane programs have, but you can add any fields to a schema.
 
 export function status() {
   return "👋 start here!";
 }
 
-// To learn more about actions in Membrane, visit: https://www.membrane.io/the-graph
+// Note: `status` is a default field that all Membrane programs have, but you can add any fields to a schema.
+// To learn more about actions and fields in Membrane, visit: https://docs.membrane.io/concepts/schema
```
Comment (juancampa) [inline index.ts]: Normally I use the word "implicit" when talking about `status`, `endpoint` and `email`. I think it better conveys how they work.

File: index.ts [hunk 4]
```diff
@@ -77,9 +75,10 @@ export function status() {
  */
 
 // Membrane comes with several built-in utils that your programs can install, like `sms` and `email`.
-// Your workspace has `sms` and `email` installed and this program includes them as dependencies. We'll cover how to add new dependencies later.
+// Your workspace has `sms` and `email` installed, and this program includes them as dependencies.
```
Comment (juancampa) [inline index.ts]: Should we use "connections" here?

File: index.ts [hunk 5]
```diff
@@ -114,11 +110,11 @@ export async function ping() {
 // Let's set up the `ping` action to run on a timer.
 // In the Membrane Explorer, click on "getting-started" then click the clock icon to the right of "ping".
 // Select the type of timer you want (e.g. "Invoke after delay...") and when you want it to run (e.g. "10s").
-// After clicking the green check to confirm, your timer will show up in the bottom right panel of the explorer.
+// After clicking the green check to confirm, your timer will appear in the CONFIG panel in the bottom right of your IDE.
```
Comment (juancampa) [inline index.ts:114]: I'm glad this is true now :)

**Review by juancampa — CHANGES_REQUESTED** [review]

File: index.ts (hunk 2 above)
Comment (pmillspaugh) [reply index.ts:55]: [T-839](https://membrane.height.app/T-839) Add back the Invoke ▶️ button in the IDE

File: index.ts (hunk 4 above)
Comment (pmillspaugh) [reply index.ts]: We could also rename `DEPENDENCIES` in the CONFIG panel

**Review by just-be-dev — APPROVED** [review]

**Pull request merged**